Makefile

```diff
@@ -187,3 +189,7 @@ native:
 
 clean:
 	$(RM) -r build
+
+publish-github-artifact: release
```

Comment:
The logic for accessing GitHub API is now spread across Actions and Makefile, which is not good. This will not work in a standalone mode anyway, unless all the required variables are passed from GHA.
What was the reason for moving it here?

Reply:
I moved it here to make the GHA script simpler, but now that the publishing happens only in one place I should be able to move it back to the script